src/eve/visitors.py

```diff
@@ -118,6 +125,10 @@ def visit(self, node: concepts.TreeNode, **kwargs: Any) -> Any:
                 if node_class is concepts.Node:
                     break
 
+        if self.previsitors:
+            for previsitor in self.previsitors:
+                previsitor(node, self, kwargs)
```

Comment:
It's not a deal-breaker but I think it would be more coherent to pass the Visitor instance (`self`) as first parameter,  which involves changing the order of args in the `PrevisitCallable` signature...

```suggestion
                previsitor(self, node, kwargs)
```

Reply:
That does look better. Thanks!